database/Seeders/DatabaseSeeder.php

```diff
@@ -12,5 +13,7 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
         $this->call(EggSeeder::class);
+
+        Role::firstOrCreate(['name' => Role::ROOT_ADMIN]);
```

Comment:
This needs to be in the migration file itself.

Reply:
Why? That's exactly what seeders are for?

Reply:
I meant in addition to, that way it gets added to everybody's. If that's already happening, then you can ignore this comment, thanks.

Reply:
If the root admin role is missing it's created automatically if needed. The seeding is only for new installs, really. So that should be fine.